hms-app/src/components/IdeaCardFoldable.tsx

```diff
@@ -50,136 +51,175 @@ export default function IdeaCardFoldable(props: IProps) {
     initialItem: -1,
   })
 
-  const { idea, type } = props
+  const [ideaData, setIdeaData] = useState({
+    errorIdeaData: false,
+    isLoadingIdeaData: true,
+    id: 'string',
+    owner: undefined,
+    hackathon: undefined,
+    participants: [],
+    title: 'string',
+    description: 'string',
+    problem: 'string',
+    goal: 'string',
+    requiredSkills: [],
+    category: undefined,
+    creationDate: 'string',
+  } as Idea)
+
+  const idea: Idea = ideaData
```

Comment:
you actually don't need this line.  You could now access ideaData directly (or rename it to idea)